evolution-chain/precompile/laos-evolution/contracts/LaosEvolution.sol

```diff
@@ -24,4 +38,18 @@ interface LaosEvolution {
     function ownerOfCollection(
         uint64 collectionId
     ) external view returns (address);
+
+    /// @notice Mint a new token
+    /// @dev Call this function to mint a new token
+    /// @param collectionId the id of the collection
+    /// @param slot the slot of the token
+    /// @param to the owner of the newly minted token
+    /// @param tokenURI the tokenURI of the newly minted token
+    /// @return the id of the newly minted token
+    function mint(
```

Comment:
suggest to maintain the name we had agreed upon on the substrate side: mintWithExternalURI